data/session-config.js

```diff
@@ -0,0 +1,24 @@
+
+const session = require('express-session');
+const KnexSessionStore = require('connect-session-knex')(session);
```

Comment:
I highly encourage using this naming convention while getting reps in, but feel free to shorten it up with a foobar-banana name of your choice if you so choose